tests/test_admin_cli.py

```diff
@@ -899,18 +903,25 @@ def setUp(self, patched_prompt) -> None:
 
     @patch("pepys_admin.snapshot_cli.input", return_value="test.db")
     def test_do_export_reference_data_postgres(self, patched_input):
+        # Delete test.db file first, in case it is hanging around from another test
+        # If we don't do this, we can get into an infinite loop
+        path = os.path.join(os.getcwd(), "test.db")
```

Comment:
I think you can use `CURRENT_DIR` variable instead of calling `os.getcwd()` again and again.